angular/projects/global/src/lib/services/config.service.ts

```diff
@@ -34,4 +34,16 @@ export class ConfigService {
   get config(): any {
     return this.configuration;
   }
+
+  public getCommunicationPackage(application, pathAPI: string) {
+    return this.httpClient.get(`${pathAPI}/config/${application}`, {});
+  }
+
+  public createCommunicationPackage(communicationPackage, pathAPI: string) {
+    if (!communicationPackage) {
```

Comment:
a `{}` will pass this check.

Reply:
Added an additional check to ensure `application` exists. The other fields can be null, API will map their lack of existence to null by default, so I don't think we need an overly strong check here.